### ISSUE TITLE
Undocumented member calculation error

### ISSUE THREAD

**Issue by slightfoot** [kf6gpe/dart-flutter-favorite#11]
Hi Ray,
Looks like override methods have to be documented at the moment. This seem's to be a bug in the `surveyor` library. I see it is suppose to ignore overridden members but this is not always the case.

https://github.com/pq/surveyor/blob/master/example/doc_surveyor/lib/src/doc_surveyor.dart#L105

Two possible fixes I can see.

1. Update the surveyor plugin to look for [`override`](https://api.dart.dev/stable/2.7.1/dart-core/override-constant.html) annotation on members.
2. Update flutter-favorite tool to use dart analyzer directly to perform same as [`public_member_api_docs`](https://dart-lang.github.io/linter/lints/public_member_api_docs.html) lint check. https://github.com/dart-lang/linter/blob/21b6699ce3b5e2ce95d6ae70254ee0443cfda7ef/lib/src/rules/public_member_api_docs.dart

**Comment by kf6gpe** [comment]
cc @csells. If I understand your comment correctly, @slightfoot, it's that we're overcounting documentation, and need to drop the count by the number of overrides that have documentation comments? That should be easy to do with the analyzer code you link. I'll try to get to it this week. That's probably easier than waiting for us to land a patch upstream.

Thanks for the issue!

**Comment by slightfoot** [comment]
Yes. Counting overridden members without  docs. And reporting the specific lines these occur. So its not just counting but the detection.

**Comment by kf6gpe** [comment]
Thanks, Simon! Do you have an example package where this occurs that we can use for internal testing? @pq may be able to help us with the fix to surveyor, it sounds like.

**Comment by pq** [comment]
If we can identify a repro, I'm more than happy to take a look.

Thanks!

**Comment by awhitford** [comment]
@pq I am seeing this issue with the package [animated_text_kit](https://pub.dev/packages/animated_text_kit).  It is triggering a missing comment for Flutter [StatefulWidget](https://api.flutter.dev/flutter/widgets/StatefulWidget-class.html)s with an overridden [createState](https://api.flutter.dev/flutter/widgets/StatefulWidget/createState.html) method.
```
  createState: fade.dart:107:3
  createState: text_liquid_fill.dart:71:3
  createState: typewriter.dart:106:3
  createState: colorize.dart:91:3
  createState: rotate.dart:111:3
  createState: typer.dart:91:3
  createState: wavy.dart:62:3
  createState: scale.dart:114:3
```
The other cases, like missing class docs, are legit.  However, these complaints about a `createState` `@override` look like they should pass.
